### DISCUSSION TITLE
Selected data has been copied to the clipboard.

### DISCUSSION THREAD

**Discussion by DrMarkX** [microsoft/azuredatastudio#24029] [Q&A]
Is there a way to stop this pop-up?  It started with the July update.

**Comment by erinstellato-ms** [comment]
@alanrenmsft I believe this is related to improvements you added around copying results...is the notification configurable?

**Reply by alanrenmsft** [reply]
unfortunately it is not configurable right now, please create an issue for it.

**Comment by erinstellato-ms** [comment]
@DrMarkX if you could create that request here: https://github.com/microsoft/azuredatastudio/issues that would be great.  Thanks!

**Reply by DrMarkX** [reply]
Done
